jep/1/README.adoc

```diff
@@ -0,0 +1,754 @@
+appropriate alternative list where discussion, review and pronouncement on the
+JEP will occur.
+
+[[accepted]]
+==== Accepting a JEP
+
+For a JEP to be "Accepted" it must meet certain minimum criteria:
+
+* It must be a clear and complete description of the proposed enhancement.
+* The enhancement must represent a net improvement.
+* The proposed implementation, if applicable, must be solid and must not complicate Jenkins unduly.
+
+Once a JEP has been accepted, the implementation must be completed.
+The Jenkins project values contribution over "talk"
+footnote:[https://jenkins.io/project/governance/#meritocracy], and as such the
+implementation is of utmost importance to moving any proposal (Standards or
+Process) forward.
+
+[[final]]
+==== Finalizing a JEP
+When the implementation is complete and incorporated into the
+appropriate "main" code repository, the JEP author will change
+the JEP's status changed to "Final".
+
+[[maintenance]]
+==== JEP Maintenance
+
+Even after a JEP reaches "Final" status, it may need to be updated.
+
+In general, Standards track JEPs are not modified after they have
+reached the Final state. Once a Standards JEP has been completed, Jenkins developer
+documentation must become the formal documentation of the expected behavior.
+
+Informational and Process JEPs may be updated over time to reflect changes
+to development practices and other details. The precise process followed in
+these cases will depend on the nature and purpose of the JEP being updated.
+
+Replaced::  [[replaced]]
+Final JEPs may eventually also be "Replaced" - superseded by a different JEP -
+rendering the original obsolete.
+This is intended for Informational JEPs, where version 2 of an API can replace version 1.
+
+==== Other JEP Outcomes
+
+Not all JEPs will be accepted and finalized.
+
+Rejected::  [[rejected]]
+A JEP can also be "Rejected". Perhaps after all is said and done it
+was not a good idea. It is still important to have a record of this
+fact.
+
+Rejected::  [[withdrawn]]
+The "Withdrawn" status is similar to "Rejected" - it means that the JEP author
+themselves has decided that the JEP is actually a bad idea,
+or agrees that a competing proposal is a better alternative.
+
+Deferred:: [[deferred]]
+A JEP can also be assigned a status of "Deferred". The JEP author or an
+editor can assign the JEP this status when no progress is being made
+on the JEP. Once a JEP is deferred, a JEP editor can re-assign it
+to draft status.
+
+Active:: [[active]]
+Some Informational and Process JEPs may also have a status of "Active" if they
+are ongoing and never meant to be completed per se. E.g. JEP 1 (this JEP).
+
+==== Updating JEP Status and Resolution
+
+Whenever a JEP status changes, the "Status" field in the JEP document must be updated.
+
+The possible paths of a JEP's status are as follows:
+
+.JEP Workflow
+image::workflow.png[JEP Workflow]
+
+When a JEP is Accepted, Rejected or Withdrawn,
```

Comment:
Not sure what it means. Maybe a mirror JIRA issue in the "JEP" project is preferable

Reply:
Updated with additional text. Resolved?

Reply:
RESOLVED